src/wp-admin/includes/class-wp-list-table.php

```diff
@@ -184,6 +184,17 @@ public function __get( $name ) {
 		if ( in_array( $name, $this->compat_fields, true ) ) {
 			return $this->$name;
 		}
+
+		_doing_it_wrong(
```

Comment:
The PHP 8.2 Dynamic Property change triggers a  `E_USER_DEPRECATED`, whereas `_doing_it_wrong()` triggers a `E_USER_NOTICE`.

I think `trigger_error()` is a better choice.

I also think it should be added to each of the property magic methods, rather than just the `__get()` and `__set()`.